docs/ISSUES.md

```diff
@@ -2,22 +2,22 @@
 
 ## Tipos de usuario
 
-- Gestor de almacen: Usuario encargado de la gestión del almacén de la empresa. N
+- Gestor de almacen: Usuario encargado de la gestión del almacén de la empresa. 
 
 
 ## ISSUES
 Descripción de los issues que se plantean en el proyecto
 
-[HU1] (https://github.com/eantoniocalo18/IV/issues/9) Determinar la periodicidad de compra de cada producto: El gestor del almacén necesita saber con qué frecuencia se vende cada producto, para poder determinar cada cuanto tiempo y cuánta cantidad comprar de nuevo.
-[HU2] (https://github.com/eantoniocalo18/IV/issues/13) Evitar que haya poco (o nada) stock de un producto: Para un gestor de almacén, sería muy útil recibir un aviso cuando el stock de un producto puesto que normalmente si un producto se acaba no se puede reponer la falta de stock instantáneamente por lo tanto es necesario realizar un nuevo pedido cuando queden pocos, no cuando se hayan acabado.
-[HU3] (https://github.com/eantoniocalo18/IV/issues/15) Evitar que el almacén esté lleno: Para un gestor de almacén, es muy importante saber cuándo está lleno el almacén ( o cerca de estar lleno) y, por lo tanto, no se pueden pedir más productos.
+[HU1](https://github.com/eantoniocalo18/IV/issues/9) Determinar la periodicidad de compra de cada producto: El gestor del almacén necesita saber con qué frecuencia se vende cada producto, para poder determinar cada cuanto tiempo y cuánta cantidad comprar de nuevo.
```

Comment:
No hagáis estos cambios para que tenga que revisar lo que no corresponde. Es darme más trabajo sin sentido.